content/blog/2023/10/kubewarden-1.8-release.md

```diff
@@ -0,0 +1,111 @@
+---
+title: Kubewarden 1.8.0 release
+authors:
+  - Víctor Cuadrado
+date: 2023-10-11
+---
+
+Today we are happy to announce the release of Kubewarden 1.8.0! 🎉🥳
+
+This is a small release, focused on [OpenTelemetry](https://opentelemetry.io/).
+
+The OpenTelemetry Protocol (OTLP) got its [first 1.0.0
+version](https://github.com/open-telemetry/opentelemetry-proto/releases/tag/v1.0.0)
+in July 2023, and the OpenTelemetry stack is becoming more stable; several
+libraries got their first 1.0.0 release, such as the [Go metric
+SDK](https://opentelemetry.io/blog/2023/otel-go-metrics-sdk-stable/) or the
+[.NET Automatic
+Instrumentation](https://opentelemetry.io/blog/2023/otel-dotnet-auto-instrumentation/).
+
+Still, the OTLP stack is not yet stable, and unannounced backwards-incompatible
```

Comment:
```suggestion
Still, the OpenTelemetry stack is not yet stable, and unannounced backwards-incompatible
```

I prefer to avoid acronyms that are probably not known by the average reader